config/jobs/cert-manager/istio-csr/cert-manager-istio-csr-presubmits.yaml

```diff
@@ -4,37 +4,57 @@ presubmits:
   - name: pull-cert-manager-istio-csr-verify
     decorate: true
     always_run: true
-    max_concurrency: 8
-    annotations:
-      testgrid-create-test-group: 'false'
-    branches:
-    - ^main$
+    labels:
+      preset-go-cache: "true"
+      preset-local-cache: "true"
     spec:
       containers:
-      - image: golang:1.21
+      - image: eu.gcr.io/jetstack-build-infra-images/make-dind:20231115-8c0910c-bookworm
         args:
+        - runner
         - make
+        - vendor-go
         - verify
         resources:
           requests:
             cpu: 1
             memory: 1Gi
 
+  - name: pull-cert-manager-istio-csr-unit
+    decorate: true
+    always_run: true
+    labels:
+      preset-go-cache: "true"
+      preset-local-cache: "true"
+    spec:
+      containers:
+      - image: eu.gcr.io/jetstack-build-infra-images/make-dind:20231115-8c0910c-bookworm
+        args:
+        - runner
+        - make
+        - vendor-go
+        - test-unit
+        resources:
+          requests:
+            cpu: 1
+            memory: 1Gi
+
   - name: pull-cert-manager-istio-csr-ca-rotation
     decorate: true
     always_run: true
-    branches:
-    - ^main$
     labels:
+      preset-go-cache: "true"
+      preset-local-cache: "true"
       preset-service-account: "true"
       preset-dind-enabled: "true"
```

Comment:
comment: It feels like if these presets are going to be common for all things using makefile-modules, then it might be worth having something like `preset-makefile-modules` in the future. Or maybe not. Certainly not a blocker for this though!